### ISSUE TITLE
Compare Weekly Reports with Case Follows is Empty

### ISSUE THREAD

**Issue by gmcressman** [ICTatRTI/coconut-analytics#328]
No data shown in report. The most likely cause is that there is no weekly data yet from MEEDS. Check to make sure report is working correctly, and script is running correctly to retrieve weekly data from MEEDS.

[CompareReport.docx](https://github.com/ICTatRTI/coconut-analytics/files/1703304/CompareReport.docx)

**Comment by scloo** [comment]
I ran the report based on the dates on his CompareReport.docx, I was able to view results. (see below )
I noticed that he was in the CSV mode, and that will not show any result. I don't think the CSV mode work.

![Screen Shot 2018-02-07 at 11.51.16 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/571a5f9b500b7b0e00eca657/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f46b7191a379ee0c2e50415e48bc30254d58eeb9cb74fce36f34985619e8d66c5821c33816aaebd417b853e9728b4f87b5ea8f45c5fa2137a5dad11b61435c52dcbd4f3fb2984cc259f34ae94d57a6103167e370366e0a.png)

**Comment by mikeymckay** [comment]
The script was running out of memory, so I fixed it about 6 hours ago. I
tweaked it to only retrieve data from 2017 onwards (it was getting earlier
data in case revisions were made) and it now runs successfully.
Unfortunately, I am now seeing blanks on the case based (right) side of
things:

https://zanzibar.cococloud.co/zanzibar/_design/analytics/index.html#reports/startDate/2018-01-01/endDate/2018-01-31/type/Compareweekly/aggregationLevel/District/mostSpecificLocationSelected/ALL

**Comment by gmcressman** [comment]
Yes, and no data on the two corresponding graphs on the dashboard. CamelCaseInstein again?

-gmc

Gordon M. Cressman
Senior Program Director | Research Computing Division | RTI International
Email: gmc@rti.org<mailto:gmc@rti.org> | Office: +1 919 541-6363 | Mobile: +1 919 271-7003 | Skype: gmcressman

Was this email too brief? Here is why<http://emailcharter.org/>
